src/sophia/utility.f

```diff
@@ -0,0 +1,69 @@
+      SUBROUTINE TOEVT()
+C************************************************************************
+C
+C     Converts data in S_PLIST common block to
+C     standard HEPEVT common block format, 
+C     and S_PLIST1 and S_CHP to SCHG common block containing
+C     charge and parent decayed particle information
+C     
+C************************************************************************
+      IMPLICIT DOUBLE PRECISION (A-H,O-Z)
+      IMPLICIT INTEGER (I-N)
+
+      SAVE
+      INTEGER PDGCOD(49)
```

Comment:
Where is this table from?

Reply:
It is the table from FUNCTION ICON_PDG_SIB(ID) in eventgen.f, which is used to convert PDG particle codes to SIBYLL particle codes